### ISSUE TITLE
tfsec failed to run

### ISSUE THREAD

**Issue by chris3ware** [aquasecurity/tfsec#1479]
tfsec has been working fine for a few weeks, however today it started to display and error stating that "tfsec failed to run"

Here is the tfsec output from vscode:

```zsh
[DEBUG][2022-02-01 22:13:02.060235 +0000 GMT m=+8.901527922] Starting scanner...

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x14dab60]

goroutine 1 [running]:

github.com/zclconf/go-cty/cty.Type.FriendlyName(...)
	/home/runner/work/tfsec/tfsec/vendor/github.com/zclconf/go-cty/cty/type.go:47
github.com/hashicorp/hcl/v2/hclsyntax.(*ConditionalExpr).Value(0xc0003ef1f0, 0xc01f7a0cd8)
	/home/runner/work/tfsec/tfsec/vendor/github.com/hashicorp/hcl/v2/hclsyntax/expression.go:623 +0x1320
github.com/aquasecurity/tfsec/internal/pkg/block.getStrings({0x1911978, 0xc0003ef1f0}, 0x10)
	/home/runner/work/tfsec/tfsec/internal/pkg/block/attribute.go:254 +0x208
github.com/aquasecurity/tfsec/internal/pkg/block.(*Attribute).ValueAsStrings(...)
	/home/runner/work/tfsec/tfsec/internal/pkg/block/attribute.go:243
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws/vpc.adaptSGRule(0xc00035da40, {0xc005380000, 0x230, 0x400})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/vpc/adapt.go:142 +0x28a
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws/vpc.(*sgAdapter).adaptSecurityGroup(0xc0031dd6d8, 0xc00045ea00, {0xc005380000, 0x230, 0x400})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/vpc/adapt.go:110 +0x793
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws/vpc.(*sgAdapter).adaptSecurityGroups(0xc0031dd6d8, {0xc005380000, 0x230
, 0x400})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/vpc/adapt.go:45 +0x11b
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws/vpc.Adapt({0xc005380000, 0x230, 0x400})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/vpc/adapt.go:16 +0x156
github.com/aquasecurity/tfsec/internal/pkg/adapter/aws.Adapt({_, _, _})
	
/home/runner/work/tfsec/tfsec/internal/pkg/adapter/aws/adapt.go:76 +0xeff
github.com/aquasecurity/tfsec/internal/pkg/adapter.Adapt({0xc005380000, 0x230, 0x400})
	/home/runner/work/tfsec/tfsec/internal/pkg/adapter/adapt.go:19 +0x4e
github.com/aquasecurity/tfsec/internal/pkg/scanner.(*Scanner).Scan(0xc006cbc660, {0xc005380000, 0x0
, 0x400})
	/home/runner/work/tfsec/tfsec/internal/pkg/scanner/scanner.go:51 +0x9f
github.com/aquasecurity/tfsec/internal/app/tfsec/cmd.glob..func1(0x1e81320, {0xc0003ee230, 0x1, 0x7
})
	/home/runner/work/tfsec/tfsec/internal/app/tfsec/cmd/root.go:114 +0x7ec
github.com/spf13/cobra.(*Command).execute(0x1e81320, {0xc00001e090, 0x7, 0x7
})
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x1e81320)
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/work/tfsec/tfsec/vendor/github.com/spf13/cobra/command.go
:902
main.main()
	/home/runner/work/tfsec/tfsec/cmd/tfsec/main.go:11 +0x25
```

**Desktop (please complete the following information):**
 - OS: macOS 10.15.7
 - Tfsec Version: v1.0.7
 - VSCode Version: v1.7.4

I was experimenting with setting: "tfsec.ignoreDownloadedModules": to false. It ran OK, but I have changed it back to true.

I have also restarted vscode and reinstalled the extension.

I tried upgrading tfsec to v1.0.8 also but the problem persists.

I should also add, tfsec runs fine from the cli.

**Comment by liamg** [comment]
This should be resolved in `v1.0.9` - it's building now.

**Comment by chris3ware** [comment]
Looks like that's fixed it. Thanks.
